config/development.js

```diff
@@ -21,13 +21,15 @@ module.exports = {
   },
 
   userToken: {
-    publicKey: '-----BEGIN PUBLIC KEY-----\n' +
+    publicKey:
```

Comment:
This does not need to be modified. 🤔

Reply:
I have not modified it. Did some copy paste or maybe my linter formatted this